### DISCUSSION TITLE
Kernel Average Misorientation (KAM) Histogram

### DISCUSSION THREAD

**Discussion by Mabroukimedcea** [mtex-toolbox/mtex#1759] [Ask Anything]
Hello everyone; 

I want to get KAM distribtion as a histogram below 
![image](https://user-images.githubusercontent.com/126521440/224057092-5a237c7d-739f-4899-a531-5e23ed5ef881.png)
 I get the maps (as below), but  I didn't find how to plot a KAM histogram 

thanks a lot for your help
![image](https://user-images.githubusercontent.com/126521440/224057808-e1dba8d1-96ca-4bc4-9db6-6a2f500bb5d5.png)

Mohamed

**Comment by M-Eddahbi** [comment]
He Mohamed,

Why did you to represent the KAM as histogram? It dosen make any sense. the KAM map is related to the misorientation varaition inside grains or through the grain boundaries to reveal if restoration mechanism (recovery and/or recrystallization is taking place. In your case, it seems that the misorientaion change up to 7 degrees occurs at the grain boundaries.
Instead, the quality image (QI) histogram is the most required, and should be accompanied the KAM image, since the QI may judge your resultas, accordin the bad or good indezation during EBSD mapping.

In my opinion, there is no special physical issue arising from your KAM map. 

Regrads
Eddahbi

**Reply by Mabroukimedcea** [reply]
Hi Eddahbi

thanks for your answer, 

so I'am studing the microstructure evolution during welding,  I want to compare the average disorientation between a measurement point and its close neighbours in the differents zone affected by the welding and these can be done using the Kernel Average Misorientation (KAM) parameter. this method is largey used also 
and this will help me to better understand the origines of differentes microstructure observed on the welding 
so figure below , the evolution of the KAM on different weld zones 
![image](https://user-images.githubusercontent.com/126521440/224158799-a45c8a78-ec47-46e0-9e55-a24edd2a85ae.png)
Mohamed
